kalite/distributed/templates/distributed/partials/_footer.html

```diff
@@ -34,7 +34,7 @@
                         {{ channel_data.footer_text }}
                     </p>
                     <p class="footer-text">
-                        <img id="creative-commons-icon" src="{% static 'images/distributed/' True %}{{ channel_data.channel_license }}.png">
+                        <img id="creative-commons-icon" src="{% static 'images/distributed/' True %}{{ channel_data.channel_license }}.png" alt="{% trans 'Creative Commons Attribution-NonCommercial-ShareAlike 3.0 United States License' %}">
```

Comment:
Works for now - but we will want the template to draw the license description from another source in the future.